satpy/tests/enhancement_tests/test_ahi.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2021 Satpy developers
+#
+# This file is part of satpy.
+#
+# satpy is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later
+# version.
+#
+# satpy is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+# A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# satpy.  If not, see <http://www.gnu.org/licenses/>.
+"""Unit testing for the AHI enhancement function."""
+
+import unittest
+import numpy as np
+import xarray as xr
+import dask.array as da
+
+
+class TestAHIEnhancement(unittest.TestCase):
+    """Test the AHI enhancement functions."""
+
+    def setUp(self):
```

Comment:
Moving towards pytest, this should be equivalent

```suggestion
class TestAHIEnhancement():
    """Test the AHI enhancement functions."""

    def setup(self):
```